### ISSUE TITLE
Manually calling handleSubmit() without passing anything not working in 3.0.2

### ISSUE THREAD

**Issue by hakanderyal** [erikras/redux-form#235]
In 2.x, I can call the `handleSubmit()` that is passed by `connectReduxForm` without any arguments.

It seems that in 3.x, it requires an event to be passed.

Maybe worth mentioning in docs or upgrade guide?

**Comment by erikras** [comment]
Hmm... You can still do
```js
<form onSubmit={handleSubmit}> // *sort of* no params (really param is event)
```
Were you just calling `handleSubmit()` directly before, and now that's not working?

**Comment by hakanderyal** [comment]
Yes, I have a form with a `textarea` field that I need to submit when `enter` is pressed. 

I was just calling `handleSubmit()` in 2.x on my keypress handler. Now it works by passing the keypress event to `handleSubmit()`

**Comment by erikras** [comment]
Okay, so I should make `handleSubmit` be okay with no params. I'll put it on the todo list.

**Comment by erikras** [comment]
Fixed in v3.0.4.
